source/core/slang-type-text-util.cpp

```diff
@@ -51,11 +51,13 @@ static const TypeTextUtil::CompileTargetInfo s_compileTargetInfos[] =
     { SLANG_SPIRV_ASM,      "spv-asm",                                          "spirv-asm,spirv-assembly", "SPIR-V assembly" },
     { SLANG_C_SOURCE,       "c",                                                "c",                        "C source code" },
     { SLANG_CPP_SOURCE,     "cpp,c++,cxx",                                      "cpp,c++,cxx",              "C++ source code" },
+    { SLANG_CPP_HEADER,     "hpp,h++,hxx",                                      "hpp,h++,hxx",              "C++ source header" },
```

Comment:
I just don't want to confuse C `.h, .inc` vs C++ headers. 
I also think that `.c++` and `.h++` is really not common, but maybe `.cc` and `.hh` can be added.

Reply:
I am fine with treating .h as cpp header because we won't be supporting c as target anytime soon. Treating cpp as the default is likely the desired behavior anyways.